src/qibolab/platform.py

```diff
@@ -5,6 +5,70 @@
 from qibolab.platforms.platform import DesignPlatform
 
 
+def create_tii_rfsoc4x2(runcard, address=None):
+    """Create platform using QICK project on the RFSoS4x2 board and Rohde Schwarz local oscillator for the TWPA
+    IPs and other instrument related parameters are hardcoded in ``__init__`` and ``setup``.
+    Args:
+        runcard (str): Path to the runcard file.
+        address (str): Address and port for the QICK board.
+            If ``None`` it will attempt to connect to TII instruments.
+    """
+    from qibolab.instruments.dummy_oscillator import (
+        DummyLocalOscillator as LocalOscillator,
+    )
+    from qibolab.instruments.rfsoc import TII_RFSOC4x2
+
+    # Create channel objects
+    channels = ChannelMap()
+    # readout
+    channels |= ChannelMap.from_names("L3-18_ro")
+    # feedback
+    channels |= ChannelMap.from_names("L2-RO")  # TODO find the real channel
+    # drive
+    channels |= ChannelMap.from_names("L3-18_qd")
+    # TWPA
+    channels |= ChannelMap.from_names("L4-26")  # TODO find the real channel
+
+    # Map controllers to qubit channels (HARDCODED)
+    # readout
+    channels["L3-18_ro"].ports = [("o0", 0)]
+    # feedback
+    channels["L2-RO"].ports = [("i0", 0)]
+    # drive
+    channels["L3-18_qd"].ports = [("o1", 1)]
+
+    # Instantiate QICK instruments
+
+    if address is None:
+        # connect to TII instruments for simulation
+        address = "192.168.2.72:6000"
+
+    controller = TII_RFSOC4x2("tii_rfsoc4x2", address)
+
+    # Instantiate local oscillators (HARDCODED) # TODO local oscillators should not be needed
+    local_oscillators = [
+        LocalOscillator("twpa", "192.168.0.35"),
+    ]
+    # Set TWPA parameters
+    local_oscillators[0].frequency = 6_511_000_000
+    local_oscillators[0].power = 4.5
+
+    # Map LOs to channels
+    channels["L4-26"].local_oscillator = local_oscillators[0]  # TODO find the real channel
+
+    design = MixerInstrumentDesign(controller, channels, local_oscillators)
+    platform = DesignPlatform("tii_rfsoc4x2", design, runcard)
+
+    # assign channels to qubits
+    qubits = platform.qubits
+    qubits[0].readout = channels["L3-18_ro"]
+    qubits[0].feedback = channels["L2-RO"]
+    qubits[0].drive = channels["L3-18_qd"]
+    channels["L4-26"].qubit = qubits[0]  # TODO find the real channel
```

Comment:
Are you using the `qubit` property for this channel? Generally it is not mandatory to set the qubit attribute of channels. It was added for convenience related to flux, but this is not relevant in your case yet.